test/extended/ldap_groups.sh

```diff
@@ -225,5 +223,12 @@ for (( i=0; i<${#schema[@]}; i++ )); do
     openshift ex sync-groups --sync-config=sync-config-dn-everywhere.yaml --confirm
 	compare_and_cleanup valid_all_ldap_sync_dn_everywhere.txt
 
+	# PRUNING
+	echo -e "\tTEST: Sync all LDAP groups from LDAP server, change LDAP UID, then prune OpenShift groups"
+	openshift ex sync-groups --sync-config=sync-config.yaml --confirm
+	oc patch group ${group2_osuid} -p '{"metadata":{"annotations":{"openshift.io/ldap.uid":""}}}'
```

Comment:
> Why is this important? If anything, having no value is better because we will for sure be tripping the out-of-bounds error check.

Because no admin is going to intentionally clear it and we're never going to create on that doesn't have it.  This is currently testing something that will never actually happen.  Not quite useless (as you pointed out), but darn close.